x-pack/plugins/fleet/server/routes/data_streams/handlers.ts

```diff
@@ -192,24 +226,7 @@ export const getListHandler: RequestHandler = async (context, request, response)
       body,
     });
   } catch (error) {
+    logger.error(error);
```

Comment:
`defaultIngestErrorHandler` also has consistent logging for the errors. Do we need this extra logging?

Reply:
leftover from debugging :D removed